oxy/yaml/mpu.yaml

```diff
@@ -0,0 +1,51 @@
+!Messages
+msgs:
+- !CANMsg
+    id: "0x500"
+    desc: "NERduino Acceleromter"
+    fields: 
+    - !CANField
+        name: "MPU/Accel/X"
+        unit: "g"
+        size: 2
+        decodings: 
+        - !BigEndian
+            bits: 8
+        format: "acceleration"
+    - !CANField
+        name: "MPU/Accel/Y"
+        unit: "g"
+        size: 2
+        decodings: 
+        - !BigEndian
+            bits: 8
+        format: "acceleration"
+    - !CANField
+        name: "MPU/Accel/Z"
+        unit: "g"
+        size: 2
+        decodings: 
+        - !BigEndian
+            bits: 8
+        format: "acceleration"
+
+- !CANMsg
+    id: "0x501"
+    desc: "MPU Status"
```

Comment:
Nit: I know that this is the MPU, but I think we might wanna rename to Ceberus Status or something because I think the RTOS application might outlive the actual PCB it is on

Reply:
Actually jk this is prob a slightly bigger discussion